docs/conf/portland/2025/news/welcome.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: June 14, 2024
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+We're excited to announce the dates for our Portland 2025 conference!
+
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We'll be back at Revolution Hall for another year,
+full of great talks, networking, and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+**We plan to launch our CFP in November**,
+and now is a great time to start thinking about topics to present!
+
+You can always look at our `previous years' talks <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year - 1 }}/speakers/>`_ for inspiration.
+
+Tickets
+-------
+
+Tickets will go on sale in {{ date.tickets_live }}.
+The ticket pricing will be as follows:
+
+* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
+* **Independent Tickets**: *{{tickets.independent.price}}*
+* **Corporate Tickets**: *{{tickets.corporate.price}}*
+
+We will be also be running our Opportunity Grants program again this year for those who need financial assistance to attend.
+We also have many opportunities for volunteers who want to help make the conference happen and get a free ticket that way.
+
```

Comment:
```suggestion
Note that you are not required to do volunteer tasks at the conference to be able to apply for an Opportunity Grant.

```
This is a really common confusion, and I think we need to stress it when these sentences are close to each other.